### ISSUE TITLE
Image classifier

### ISSUE THREAD

**Issue by harrisonpim** [wellcometrust/platform#1958]
Extends #1833 
By extracting some some categorical metadata on a subset of works, training an image classifier to point to domain-specific categories should be simple. We should then be able to make 'educated guesses' about the class of a work in cases where specific metadata is missing

- [x] pre-trained vgg16 (or similar) with only final dense layers set to trainable
- [x] fully-trainable custom architecture with fewer conv layers, more dropout

- [x] single target class
- [x] multi-class
